src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -53,6 +53,23 @@ namespace {
 
         return label.getElement();
     };
+
+    bool checkMessage(const QString &userName, MessagePtr message)
+    {
+        bool isSubscription = message->flags.has(MessageFlag::Subscription) &&
+                              message->loginName == "" &&
```

Comment:
```suggestion
                              message->loginName.isEmpty() &&
```

Reply:
This and your other suggestions are fixed in 91c3889 :+1: